force-matching/scripts/translate_table.py

```diff
@@ -48,16 +47,21 @@
 		f = float( data[2].strip() )
 
 
+	u = u / 4.184
```

Comment:
This is equivalent to prior but was probably changed to make clear that this is always the conversion